f5_openstack_agent/lbaasv2/drivers/bigip/ftp_profile.py

```diff
@@ -0,0 +1,106 @@
+# coding=utf-8
+# Copyright (c) 2016-2018, F5 Networks, Inc.
```

Comment:
fix copyright year?

Reply:
> fix copyright year?

not in this commit? a lot of copyright year
```
[stack@neutron-server-1 bigip]$ grep -rn Copyright
exceptions.py:1:# Copyright (c) 2014-2018, F5 Networks, Inc.
network_service.py:1:# Copyright (c) 2014-2018, F5 Networks, Inc.
utils.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
lbaas_service.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
esd_filehandler.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
ssl_profile.py:2:# Copyright (c) 2016-2018, F5 Networks, Inc.
agent_manager_lite.py:3:# Copyright (c) 2020, F5 Networks, Inc.
cluster_manager.py:1:# Copyright (c) 2014-2018, F5 Networks, Inc.
fdb_connector_ml2.py:1:# Copyright (c) 2014-2018, F5 Networks, Inc.
virtual_address.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
vcmp.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
service_adapter.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
stat_helper.py:1:# Copyright (c) 2014-2018, F5 Networks, Inc.
test/test_service_adapter.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
test/certs_sample/samples.py:2:# Copyright (c) 2016-2018, F5 Networks, Inc.
test/conftest.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_agent_manager.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_virtual_address.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
test/test_icontrol_driver.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/mock_builder_base_class.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_stat_helper.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
test/test_vlan.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
test/test_vcmp.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
test/test_exceptions.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_lbaas_builder.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
test/test_plugin_rpc.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_network_service.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_listener_services.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_l2_service.py:3:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_ssl_profile.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
test/test_icontrol_driver_opts.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_pool_services.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_agent_manager_LbaasAgentManager.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_esd_filehandler.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_cert_parser.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
test/test_network_helper.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_utils.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
test/class_tester_base_class.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
test/test_l7policy_service.py:2:# Copyright (c) 2017,2018, F5 Networks, Inc.
resource_manager.py:2:# Copyright (c) 2020, F5 Networks, Inc.
refersh_esd.py:3:# Copyright (c) 2014-2018, F5 Networks, Inc.
listener_service.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
lbaas_builder.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
snats.py:1:# Copyright (c) 2014-2018, F5 Networks, Inc.
icontrol_driver.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
system_helper.py:1:# Copyright (c) 2014-2018, F5 Networks, Inc.
pool_service.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
l7policy_service.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
l2_service.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
service_launcher.py:3:# Copyright (c) 2014-2018, F5 Networks, Inc.
vlan_binding.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
barbican_cert.py:2:# Copyright (c) 2016-2018, F5 Networks, Inc.
constants_v2.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
lbaas_driver.py:1:# Copyright (c) 2014-2018, F5 Networks, Inc.
ftp_profile.py:2:# Copyright (c) 2016-2018, F5 Networks, Inc.
agent_manager.py:3:# Copyright (c) 2016-2018, F5 Networks, Inc.
l7policy_adapter.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
fdb_connector.py:1:# Copyright (c) 2014-2018, F5 Networks, Inc.
tenants.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
resource_helper.py:2:# Copyright (c) 2016-2018, F5 Networks, Inc.
l3_binding.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
selfips.py:2:# Copyright (c) 2014-2018, F5 Networks, Inc.
plugin_rpc.py:2:# Copyright (c) 2016-2018, F5 Networks, Inc.
agent.py:2:# Copyright (c) 2016-2018, F5 Networks, Inc.
network_helper.py:1:# Copyright (c) 2014-2018, F5 Networks, Inc.
```